packages/reports/blueprints/navi-reports/index.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+module.exports = {
+  normalizeEntityName() {}, // no-op since we're just adding dependencies
+
+  afterInstall() {
+    return this.addAddonsToProject({
+      packages: [{ name: 'navi-core' }]
```

Comment:
Is this to force the core blueprint to run when running reports?

Reply:
This allows you to simply do `ember install navi-reports` and get navi-core.